cmd/podman/containers/exists.go

```diff
@@ -12,10 +12,10 @@ var (
 	containerExistsDescription = `If the named container exists in local storage, podman container exists exits with 0, otherwise the exit code will be 1.`
 
 	existsCommand = &cobra.Command{
-		Use:   "exists CONTAINER",
+		Use:   "exists [flags] CONTAINER",
```

Comment:
You've 'flags' here and 'options' in the man page, I think it should be options everywhere.
```suggestion
		Use:   "exists [options] CONTAINER",
```

Reply:
Let's do this in a different PR.  We exchange flags and options all over the place, and one of the tests is looking for flags.

Reply:
grep -r Use.*'\[flags' cmd | wc -l
82